AardvarkLoggingUI.podspec

```diff
@@ -0,0 +1,17 @@
+Pod::Spec.new do |s|
+  s.name     = 'AardvarkLoggingUI'
+  s.version  = '1.0.0'
+  s.license  = 'Apache License, Version 2.0'
+  s.summary  = 'Aardvark is a library that makes it dead simple to create actionable bug reports.'
+  s.homepage = 'https://github.com/square/Aardvark'
+  s.authors  = 'Square'
+  s.source   = { :git => 'https://github.com/square/Aardvark.git', :tag => "AardvarkLoggingUI/#{ s.version.to_s }" }
+
+  s.swift_version = '4.0'
```

Comment:
Might be nice to jump straight to 5 while we're making breaking changes (can be done in a separate PR before we release this new podspec)

Reply:
Yep, good idea. I'll update all of the specs to Swift 5 in a follow-up PR.